### PR TITLE
documentation of Sersic profile fixed

### PR DESCRIPTION
**Pull request by sibirrer**
Thanks to @nanz6

### CODE REVIEW

File: lenstronomy/LensModel/Profiles/base_profile.py [hunk 1]
```diff
@@ -75,7 +75,7 @@ def density_lens(self, *args, **kwargs):
 
     def mass_3d_lens(self, *args, **kwargs):
         """Mass enclosed a 3d sphere or radius r given a lens parameterization with
```
Comment (ajshajib) [inline lenstronomy/LensModel/Profiles/base_profile.py:77]:
```suggestion
        """Mass enclosed within a 3D sphere or radius r given a lens parameterization with
```

File: lenstronomy/LightModel/Profiles/sersic.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ class SersicElliptic(SersicUtil):
 
     .. math::
 
-        I(R) = I_0 \\exp \\left[ -b_n (R/R_{\\rm Sersic})^{\\frac{1}{n}}\\right]
+        I(R) = I_0 \\exp \\left[ -b_n \\left[(R/R_{\\rm Sersic})^{\\frac{1}{n}}-1\\right]\\right]
```
Comment (ajshajib) [inline lenstronomy/LightModel/Profiles/sersic.py:70]:
```suggestion
        I(R) = I_{\\rm e} \\exp \\left[ -b_n \\left[(R/R_{\\rm Sersic})^{\\frac{1}{n}}-1\\right]\\right]
```

File: lenstronomy/LightModel/Profiles/sersic.py [hunk 3]
```diff
@@ -16,7 +16,7 @@ class Sersic(SersicUtil):
     """This class contains functions to evaluate a spherical Sersic function.
 
     .. math::
-        I(R) = I_0 \\exp \\left[ -b_n (R/R_{\\rm Sersic})^{\\frac{1}{n}}\\right]
+        I(R) = I_0 \\exp \\left[ -b_n \\left[(R/R_{\\rm Sersic})^{\\frac{1}{n}}-1\\right]\\right]
```
Comment (ajshajib) [inline lenstronomy/LightModel/Profiles/sersic.py:19]:
```suggestion
        I(R) = I_{\\rm e} \\exp \\left[ -b_n \\left[(R/R_{\\rm Sersic})^{\\frac{1}{n}}-1\\right]\\right]
```

File: lenstronomy/LightModel/Profiles/sersic.py [hunk 4]
```diff
@@ -67,7 +67,7 @@ class SersicElliptic(SersicUtil):
 
     .. math::
 
-        I(R) = I_0 \\exp \\left[ -b_n (R/R_{\\rm Sersic})^{\\frac{1}{n}}\\right]
+        I(R) = I_{\\rm e} \\exp \\left[ -b_n \\left[(R/R_{\\rm Sersic})^{\\frac{1}{n}}-1\\right]\\right]
```
Comment (ajshajib) [inline lenstronomy/LightModel/Profiles/sersic.py:70]:
```suggestion
        I(R) = I_{\\rm e} \\exp \\left( -b_n \\left[(R/R_{\\rm Sersic})^{\\frac{1}{n}}-1\\right]\\right)
```

File: lenstronomy/LightModel/Profiles/sersic.py [hunk 5]
```diff
@@ -16,7 +16,7 @@ class Sersic(SersicUtil):
     """This class contains functions to evaluate a spherical Sersic function.
 
     .. math::
-        I(R) = I_0 \\exp \\left[ -b_n (R/R_{\\rm Sersic})^{\\frac{1}{n}}\\right]
+        I(R) = I_{\\rm e} \\exp \\left[ -b_n \\left[(R/R_{\\rm Sersic})^{\\frac{1}{n}}-1\\right]\\right]
```
Comment (ajshajib) [inline lenstronomy/LightModel/Profiles/sersic.py:19]:
```suggestion
        I(R) = I_{\\rm e} \\exp \\left( -b_n \\left[(R/R_{\\rm Sersic})^{\\frac{1}{n}}-1\\right]\\right)
```

File: lenstronomy/LensModel/Profiles/sersic.py [hunk 6]
```diff
@@ -13,7 +13,7 @@ class Sersic(SersicUtil, LensProfileBase):
     this class contains functions to evaluate a Sersic mass profile: https://arxiv.org/pdf/astro-ph/0311559.pdf
 
     .. math::
-        \\kappa(R) = \\kappa_{\\rm eff} \\exp \\left[ -b_n (R/R_{\\rm Sersic})^{\\frac{1}{n}}\\right]
+        \\kappa(R) = \\kappa_{\\rm eff} \\exp \\left[ -b_n \\left[(R/R_{\\rm Sersic})^{\\frac{1}{n}}-1\\right]\\right]
```
Comment (ajshajib) [inline lenstronomy/LensModel/Profiles/sersic.py:16]:
```suggestion
        \\kappa(R) = \\kappa_{\\rm eff} \\exp \\left( -b_n \\left[(R/R_{\\rm Sersic})^{\\frac{1}{n}}-1\\right]\\right)
```

**Review by ajshajib — APPROVED** [review]

**Comment by sibirrer** [comment]
Thank you very much @ajshajib !

**Pull request merged**